src/card/card-directive.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.card:pfCard
+ * @restrict A
+ * @element ANY
+ * @param {headtitle=} Title for the card - required
+ * @param {subtitle=} Subtitle for the card - optional
+ *
+ * @description
+ * Directive for easiliy displaying a html with transcluded content
```

Comment:
Some typos, I think this should read:

```
Directive for easily displaying html with transcluded content
```

or even "...displaying a card with..."